ios/FluentUI/Vnext/Design Token System/ListTokens.swift

```diff
@@ -55,14 +94,16 @@ public class MSFListTokens: MSFTokensBase, ObservableObject {
     @Published public var disclosureSize: CGFloat!
     @Published public var horizontalCellPadding: CGFloat!
     @Published public var iconInterspace: CGFloat!
-    @Published public var iconSize: CGFloat!
+    @Published public var leadingViewSize: CGFloat!
+    @Published public var trailingItemSize: CGFloat!
+
     @Published public var subtitleFont: UIFont!
     @Published public var textFont: UIFont!
 
-    var iconStyle: MSFListIconStyle!
+    var cellLeadingViewSize: MSFListCellLeadingViewSize!
```

Comment:
This should be a @Published public var too.
If the theme is changed and the Tokens class resets this property, the controls need to be updated automatically.